spec/intl.html

```diff
@@ -1441,10 +1462,173 @@ <h1>get Temporal.ZonedDateTime.prototype.eraYear</h1>
             1. Perform ? RequireInternalSlot(_zonedDateTime_, [[InitializedTemporalZonedDateTime]]).
             1. Let _calendar_ be _zonedDateTime_.[[Calendar]].
             1. Let _method_ be ? Get(_calendar_, *"eraYear"*).
-            1. Return ? Call(_method_, _calendar_, « _zonedDateTime_ »).
+            1. Let _result_ ? Call(_method_, _calendar_, « _zonedDateTime_ »).
+            1. If _result_ is not *undefined*, then
+              1. Set _result_ to ? ToIntegerOrInfinity(_result_).
+            1. Return _result_.
           </emu-alg>
         </emu-clause>
       </emu-clause>
     </ins>
+
+    <emu-clause id="sec-abstracts">
+      <h1>Abstract Operations</h1>
+
+      <emu-clause id="sup-temporal-preparetemporalfields">
```

Comment:
Another thing that I might not have fully realized when you asked me about it earlier. Should the full table with era and eraYear just be in the main text? The decision at https://github.com/tc39/proposal-temporal/issues/1229#issuecomment-763056041 seems to suggest it should.

On the other hand, if not, should 402 override PreparePartialTemporalFields as well?

Reply:
@Ms2ger 's implementation of era and eraYear methods put it all in `intl` only, so I thought references to those should be in intl as well. Should those be moved out of intl into all non-intl prototypes?

Reply:
Yes, the _methods_ themselves should be in `Intl` only, but I don't see a way to avoid mentioning the _names_ of the properties in 262 and still fulfill the requirements.